src/server/rdb_save.cc

```diff
@@ -34,8 +34,9 @@ extern "C" {
 ABSL_FLAG(int, compression_mode, 2,
           "set 0 for no compression,"
           "set 1 for single entry lzf compression,"
-          "set 2 for multi entry zstd compression on df snapshot and single entry on rdb snapshot");
-ABSL_FLAG(int, zstd_compression_level, 2, "Compression level to use on zstd compression");
+          "set 2 for multi entry zstd compression on df snapshot and single entry on rdb snapshot,"
```

Comment:
I prefer that lz4 were the default compressor

Reply:
yes but still not supported, I will put that in default after I will finish the saveer and loader. If I put it as default now the python test will fail